src/com_tjvendors/admin_language/en-GB/en-GB.com_tjvendors.ini

```diff
@@ -30,7 +30,7 @@ COM_TJVENDORS_TO_FILTER="To %s"
 COM_TJVENDORS_VIEW_FILE="[View File]"
 COM_TJVENDORS_ITEMS_SUCCESS_DUPLICATED="Items successfully duplicated"
 COM_TJVENDORS_TITLE_VENDORS_FEES="Fees List"
-COM_TJVENDORS_FEES_PERCENT_ERROR="Invalid Percent Commission: Percent commission should be less than 100"
+COM_TJVENDORS_FEES_PERCENT_ERROR="Percent commission should be less than 100, or if you want to set 100 all the money will come to the site owner account."
```

Comment:
Change as
`Percent commission should be less than 100 otherwise all the money will be deposited against site owner's account.` @aishwaryab2

Reply:
okay